templates/types/multiagent/fastapi/app/examples/orchestrator.py

```diff
@@ -1,27 +1,35 @@
 from typing import List, Optional
-from app.agents.single import FunctionCallingAgent
+
 from app.agents.multi import AgentOrchestrator
+from app.agents.single import FunctionCallingAgent
+from app.examples.publisher import create_publisher
 from app.examples.researcher import create_researcher
-
 from llama_index.core.chat_engine.types import ChatMessage
 
 
 def create_orchestrator(chat_history: Optional[List[ChatMessage]] = None):
     researcher = create_researcher(chat_history)
     writer = FunctionCallingAgent(
         name="writer",
-        role="expert in writing blog posts",
-        system_prompt="""You are an expert in writing blog posts. You are given a task to write a blog post. Don't make up any information yourself. If you don't have the necessary information to write a blog post, reply "I need information about the topic to write the blog post". If you have all the information needed, write the blog post.""",
+        description="expert in writing blog posts, need information and images to write a post",
+        system_prompt="""You are an expert in writing blog posts. 
+        You are given a task to write a blog post. Don't make up any information yourself. 
+        If you don't have the necessary information to write a blog post, reply "I need information about the topic to write the blog post". 
+        If you need to use images, reply "I need images about the topic to write the blog post". Don't use any dummy images made up by you.
+        If you have all the information needed, write the blog post.""",
         chat_history=chat_history,
     )
     reviewer = FunctionCallingAgent(
         name="reviewer",
-        role="expert in reviewing blog posts",
+        description="expert in reviewing blog posts, need a written blog post to review",
         system_prompt="""You are an expert in reviewing blog posts. You are given a task to review a blog post. Review the post and fix the issues found yourself. You must output a final blog post.
+        A post must include at lease one valid image, if not, reply "I need images about the topic to write the blog post". An image URL start with example or your website is not valid.
```

Comment:
`An image URL start with example or your website is not valid.` - don't understand that command :D